lib/config.py

```diff
@@ -67,6 +74,8 @@ def _get_parameters_backup():
 def _get_parameters_restore():
     return merge_dict(parameters, parameters_restore)
 
+def _get_parameters_restore_optional():
+    return merge_dict(parameters, parameters_restore_optional)
```

Comment:
This line including 'iaas' and 'type'. So, 
https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/34/files#diff-f6edbacd8464faaf2fcb0f1121f6f7cbR99 check makes 'iaas' and 'type' optional.

You can do only

`
return parameters_restore_optional;
`

Reply:
yeah, good point. I missed it. will change it.